tests/cpp/test_matmul.cpp

```diff
@@ -3706,7 +3708,26 @@ TEST_F(HopperMatmulTest, HSH_NT_128BSwizzle) {
 
   auto tv3 = castOp(DataType::Half, tv2);
 
-  fusion.addOutput(tv3);
+  TensorView* tv4 = nullptr;
+  if (use_smem_epilogue) {
```

Comment:
Nit: you can unconditionally addOutput here then if using smem epilogue use cacheBefore to create the set op